### PR TITLE
Re: add ios fix

### PR DESCRIPTION
**Pull request by yeomann**
Re: added ios fix under sweetAlert.init function

### CODE REVIEW

File: src/sweetalert2.js [hunk 1]
```diff
@@ -1176,6 +1176,52 @@ sweetAlert.init = function() {
...
+   *  @summary IOS input focus fix
+   *  @var {iosfix}, {head} 
+   *  @description creating <style> tag with ios on focus fix
+   *  @author yeomann
+  */
+  var iosfix;
+  iosfix = document.createElement('style');
+  iosfix.type = "text/css";
+  iosfix.id = "iosFix"
+  iosfix.innerHTML = "html,body {height: 100%; position: relative; }";
```
Comment (limonte) [inline src/sweetalert2.js]: I just tested and this method doesn't work on iPad Air 2 (iOS 9.3)

Comment (yeomann) [inline src/sweetalert2.js]: do you mind addition of `overflow: hidden;` under html,body{}, i am pretty sure it will work.
PS: it worked for me on my iphone. how come it didn't work on your ipad! interesting.

Comment (limonte) [inline src/sweetalert2.js]: from my side I have now idea why this code should work :) Can you please attach the gif of swal2 working correctly on your iphone?

Comment (yeomann) [inline src/sweetalert2.js]: OR do you mind adding yourself  like this 
`iosfix.innerHTML = "html,body {height: 100%; position: relative; overflow: hidden; }"; `
and see if this solves the issue?

Comment (limonte) [inline src/sweetalert2.js]: > iosfix.innerHTML = "html,body {height: 100%; position: relative; overflow: hidden; }";
and see if this solves the issue?

doesn't work.

**Comment by Tonkpils** [comment]
@limonte can we keep this open if @yeomann is open to working on the correct fix? we should let contributors keep a bit of ownership on their PRs

**Comment by limonte** [comment]
@Tonkpils you're right :+1:

**Comment by yeomann** [comment]
@limonte, The previous fix was tested on 9.1. New commit is tested on 9.3.2, again on my iPhone. Please have a look if all looks good on your iPad? Thanks

**Comment by yeomann** [comment]
@limonte  @samturrell  any news on new commit guys? or @Tonkpils could you mind have a look at the new commit?

**Comment by limonte** [comment]
@yeomann even with 2nd commit doesn't work on my iPad (iOS 9.3, Chrome/Safari). You can test it by yourself on https://limonte.github.io/sweetalert2/

**Comment by limonte** [comment]
@yeomann I temporarily reverted https://limonte.github.io/sweetalert2/ to stable, let me know when you'll be ready to test your PR, so I can enable it again on https://limonte.github.io/sweetalert2/

**Comment by yeomann** [comment]
@limonte i will be testing on ipad today. Lets see what wrong on ipad!

**Comment by yeomann** [comment]
Tested on IOS 9.3.1 Ipad mini retina
gif image

**Comment by limonte** [comment]
Thanks for the gif @yeomann ! As you can see, on `focusout` event page below the modal is scrolled to top, which is unexpected behavoiur.

**Comment by yeomann** [comment]
@limonte  yes i observed it, not sure what causes it to go on top of screen. but i also observed that my solution simply blocks the touch ability;hence no touch move on focus. I believe there should be a bit of room for touch to go up and down. Specially for smaller screens or so as whole container is not move-able/seen-able.

**Comment by limonte** [comment]
@yeomann The main problem in iOS for me is that user after interaction with the modal isn't there where he was at the moment of launching swal2, 

Example: there's a pretty large page with height 2000px with the button which launches swal2 modal with the input field. The button is placed in the footer, e.g. at the bottom of the page.

Currently, after user's interaction with modal, the page will  be automatically scrolled to the middle of the page. This is incorrect iOS behaviour.

With your PR we're getting the same problem - after user's interaction with modal, he is not anymore at the same scroll position. But instead of autoscrolling to the middle, the page will be scrolled to the top.

In other words with additional code, we have not solved the main problem.

**Comment by yeomann** [comment]
@limonte  true. i agree! i didn't expect this PR to be merged as well as i am also not happy with the iPad results. I will check what can i do for that.

**Comment by limonte** [comment]
Thank you, @yeomann! I really appreciate your effort on this issue.

**Comment by KHam0425** [comment]
I don't have an iPad that I can test this on, but I'm watching the behavior.  It appears that it is relocating the modal to align the top of the modal to a pixel value of the viewport that isn't accounting for whether or not it needs the extra space for the keyboard to be available.  

If you look at the distance between the top edge of the viewport and the top of the modal, it seems to be an almost identical distance in both portrait and landscape layout.  Is it possible to align the middle (vertical and horizontal center) of the modal with the middle of the display?  Perhaps then it can be adjusted to re-center when the keyboard raises and lowers, which would keep the modal centered in the viewable area.

I hope this doesn't sound like gibberish, but it may require dynamically re-positioning the modal so it tracks with the "usable" screen space (aka, not covered by keyboard).

**Comment by limonte** [comment]
I'm closing this PR due to inactivity. @yeomann reopen if needed.

**Comment by yeomann** [comment]
@limonte really would like to solve this. But i am sorry don't have ipad or access to ipad at the moment.  Don't know how can i convey this solution. Will try to get a hold of one by this weekend, Hope so.
